cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -34,7 +37,33 @@
             'link_url': '#'
         }) }}
     </div>
-    <div class="block">
+    <div class="o-full-width-text">
+        {{ full_width_text.render( 'sublanding-page-1/_template-prototype-full-width-content.html' ) }}
+        <div class="m-inset">
+            {{ related_links.render( [
+                ['#', 'Related Link 1'],
```

Comment:
Mmm that'll disappear after backend gets to it huh? I guess it only matters in whether the implementation here has any effect on how @kave @kurtw implement the backend because there are other places that break links into an object so we'd want those the same. Otherwise seems fine to leave it.